public/app/features/variables/state/actions.test.ts

```diff
@@ -439,4 +453,80 @@ describe('shared actions', () => {
       });
     });
   });
+
+  describe('changeVariableMultiValue', () => {
+    describe('when changeVariableMultiValue is dispatched for variable with multi enabled', () => {
+      it('then correct actions are dispatched', () => {
+        const custom = customBuilder()
+          .withId('custom')
+          .withMulti(true)
+          .withCurrent(['A'], ['A'])
+          .build();
+
+        reduxTester<{ templating: TemplatingState }>()
+          .givenRootReducer(getTemplatingRootReducer())
+          .whenActionIsDispatched(addVariable(toVariablePayload(custom, { global: false, index: 0, model: custom })))
+          .whenActionIsDispatched(changeVariableMultiValue(toVariableIdentifier(custom), false), true)
+          .thenDispatchedActionsPredicateShouldEqual(actions => {
+            const [changeMultiAction, changeCurrentAction] = actions;
+            const expectedNumberOfActions = 2;
+
+            expect(changeMultiAction).toEqual(
+              changeVariableProp(toVariablePayload(custom, { propName: 'multi', propValue: false }))
+            );
+            expect(changeCurrentAction).toEqual(
+              changeVariableProp(
+                toVariablePayload(custom, {
+                  propName: 'current',
+                  propValue: {
+                    value: 'A',
+                    text: 'A',
+                    selected: true,
+                  },
+                })
+              )
+            );
+
+            return actions.length === expectedNumberOfActions;
+          });
```

Comment:
nit: you might consider using this form instead:
```suggestion
          .thenDispatchedActionsEqual(changeMultiAction).toEqual(
              changeVariableProp(toVariablePayload(custom, { propName: 'multi', propValue: false })),
              changeVariableProp(toVariablePayload(custom, {
                  propName: 'current',
                  propValue: {
                    value: 'A',
                    text: 'A',
                    selected: true,
                  },
                })),
          );
```